src/Tizen.Applications.Common/Interop/Interop.Libc.cs

```diff
@@ -17,11 +17,21 @@
 using System;
 using System.Runtime.InteropServices;
 
+using Tizen.Internals;
+
 internal static partial class Interop
 {
     internal static partial class Libc
     {
         [DllImport(Libraries.Libc, EntryPoint = "free", CallingConvention = CallingConvention.Cdecl)]
         internal static extern int Free(IntPtr ptr);
+
+        [NativeStruct("struct timespec", Include = "time.h")]
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct TimeStamp
+        {
+            public uint sec;
+            public uint nsec;
```

Comment:
Is using uint okay?
uint size is unsigned-32bit integer.
The size of the nsec of the struct timespec will be 8bytes in the 64-bit machine.

Reply:
You should have two pair of struct and interop api for 32bit and 64bit
``` c#
[DllImport(Libraries.RpcPort, EntryPoint = "rpc_port_parcel_header_get_timestamp")]
 internal static extern ErrorCode GetTimeStamp32(IntPtr parcelHeaderHandle, ref Libc.TimeStamp32 time);

[DllImport(Libraries.RpcPort, EntryPoint = "rpc_port_parcel_header_get_timestamp")]
internal static extern ErrorCode GetTimeStamp64(IntPtr parcelHeaderHandle, ref Libc.TimeStamp64 time);
```

And you can check byte length on runtime like below
``` c#
bool is32bit = IntPtr.Size == 4;
bool is64bit = IntPtr.Size == 8;
```

Reply:
@WonyoungChoi Could you give a guide to prepare 64bit system? if not enough it.

Reply:
I changed to nint.
According to the reference, the type follows the system architecture.
https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/builtin-types/nint-nuint
Please review them. Thanks

Reply:
awsome!

but I think you need to update .csproj to use language 9.0 spec like below
``` .csproj
  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <LangVersion>9.0</LangVersion>
  </PropertyGroup>
```

Is there no error on build?